nixos/modules/services/desktops/gnome3/gnome-settings-daemon.nix

```diff
@@ -38,8 +38,22 @@ in
 
     environment.systemPackages = [ cfg.package ];
 
+    systemd.packages = [ cfg.package ];
```

Comment:
So this module is shared by mate and pantheon, I don't think mate will have this config and pantheon still has to use an older g-s-d. Can we move this config to `services.gnome3.core-shell`, with a comment why it isn't in the g-s-d module?

Reply:
Yeah, that makes sense.